src/components/TextArea/AutoFocusTextArea.js

```diff
@@ -41,7 +41,7 @@ AutoFocusTextArea.propTypes = {
   value: PropTypes.string,
   placeholder: PropTypes.string,
   onChangeText: PropTypes.func,
-  style: PropTypes.object,
+  style: PropTypes.object
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `maxLength` prop seems to be missing in the PropTypes validation. It was mentioned in the PR summary that a `maxLength` prop set to 350 has been added to the `AutoFocusTextArea` component, but it's not reflected in the code changes. Please add it.
```diff
   value: PropTypes.string,
   placeholder: PropTypes.string,
   onChangeText: PropTypes.func,
+  maxLength: PropTypes.number,
   style: PropTypes.object
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->